src/Workload/Microsoft.NET.Sdk.Maui/WorkloadManifest.in.json

```diff
@@ -31,7 +31,13 @@
           "Microsoft.Maui.Controls.Sdk",
           "Microsoft.Maui.Extensions",
           "Microsoft.Maui.Resizetizer.Sdk",
-          "Microsoft.Maui.Templates"
+          "Microsoft.Maui.Templates",
+          "Microsoft.Maui.Core.Ref.any",
+          "Microsoft.Maui.Core.Runtime.any",
+          "Microsoft.Maui.Controls.Ref.any",
+          "Microsoft.Maui.Controls.Runtime.any",
+          "Microsoft.Maui.Essentials.Ref.any",
+          "Microsoft.Maui.Essentials.Runtime.any"
```

Comment:
I am not sure if these should be in a separate workload (maui-ref) or it is fine here (maui-core)

Reply:
I think they have to be in `maui-core`, because even if you installed `maui-android` (or any single platform) by itself, you'd still expect `net6.0` projects to work.